packages/block-editor/src/components/inner-blocks/use-nested-settings-update.js

```diff
@@ -108,11 +115,29 @@ export default function useNestedSettingsUpdate(
 		}
 
 		if ( __experimentalDefaultBlock !== undefined ) {
-			newSettings.__experimentalDefaultBlock = __experimentalDefaultBlock;
+			deprecated( '__experimentalDefaultBlock', {
+				alternative: 'defaultBlock',
+				since: '6.3',
+				version: '6.4',
```

Comment:
Hey, should we follow up on these deprecations and act on them (remove or extend) as we've reached the version.